python/2024.yocolor/setup.py.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
This is test data, not part of the sample.

Reply:
Removed in [`343c293` (#3)](https://github.com/chainguard-dev/bincapz-samples/pull/3/commits/343c29382ec52fdd820832cd5bff697723e9a1a0).